_includes/getting-started-template.html

```diff
@@ -61,6 +61,36 @@ <h1 id="templates" class="page-header">Templates</h1>
 
 </div>
 
+<h3><a name="user-content-commonjs-support" class="anchor" href="#commonjs-support"><span class="octicon octicon-link"></span></a>CommonJS Support</h3>
+
+<p>If you use a CommonJS build system (such as <a href="https://webpack.github.io/">Webpack</a> or <a href="http://browserify.org/">Browserify</a>), we recommend only loading the controls you need. You can also load all the controls by including the npm package.</p>
+
+<h4>Working with Browserify</h4>
+
+<p>This should work out of the box with Browserify.</p>
+
+<h4>Working with webpack</h4>
+
+{% highlight js %}
+npm install --save-dev imports-loader
+{% endhighlight %}
+
+{% highlight js %}
+// webpack 2 config
+// add as a rule to module.rule
```

Comment:
I think we should push for progression. If people need to integrate with webpack 1, they can look it up?